vllm_gaudi/extension/features.py

```diff
@@ -35,6 +35,11 @@ def get_user_flags():
         # Non-vllm flags that are also important to print
         Env('EXPERIMENTAL_WEIGHT_SHARING', str),
         Env('PT_HPU_WEIGHT_SHARING', str),
+
+        # Sliding window flags
+        Env('PT_HPU_SDPA_QKV_SLICE_MODE_FWD', boolean),
+        Env('PT_HPU_SDPA_BC_FACTOR', int),
```

Comment:
what is BC_factor?

Reply:
These are one of the complex_guid lib internal slice_size. We might not need to look at this more in the latest docker. Need more investigation.